go/extractor/cli/go-autobuilder/go-autobuilder.go

```diff
@@ -480,9 +480,17 @@ func installDependencies(workspace project.GoWorkspace) {
 
 		// get dependencies for all modules
 		for _, module := range workspace.Modules {
+			path := filepath.Dir(module.Path)
+
+			if util.DirExists(filepath.Join(path, "vendor")) {
```

Comment:
Looks like this happens on both the existing module and the temporary module for legacy project paths. What are the expected consequences in each case?